BlackMarket/Views/TabBar/TabbedItems.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  TabbedItems.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 26/09/2023.
+//
+
+enum TabbedItems: Int, CaseIterable {
```

Comment:
Enums are commonly named in singular.
```suggestion
enum TabItem: Int, CaseIterable {
```